data/siggraph-2020/10.1145-3386569.3392430.json

```diff
@@ -0,0 +1,97 @@
+[
+    {
+        "Variant name": "Ubuntu 18.04",
+        "Reviewer name": "Pierre Bourquat <pierre.bourquat@liris.cnrs.fr>",
+
+        "Is master variant (boolean)": true,
+
+        "Is variant deprecated (boolean)": false,
+
+        "Operating system for the test": "Ubuntu 18.04",
+
+        "Title": "Quad-mesh based isometric mappings and developable surfaces",
+
+        "DOI": "10.1145/3386569.3392430",
+
+        "Year": 2020,
+
+        "ACM Keywords": [
+            "Shape modeling",
+            "Optimization algorithms"
+        ],
+
+        "Topic {Rendering, Animation and Simulation, Geometry, Images, Virtual Reality, Fabrication}": "Geometry",
+
+        "Co-authors from academia (boolean)": true,
+
+        "Co-authors from industry (boolean)": false,
+
+        "ACM Open Access (boolean)": false,
+
+        "PDF on the authors' webpage / institution (boolean)": true,
+
+        "PDF on Arxiv or any openarchive initiatives (boolean)": false,
+        "Arxiv/OAI page URL": "",
+
+        "PDF URL": "https://www.dmg.tuwien.ac.at/geom/ig/publications/isometries/isometries.pdf",
+
+        "Project URL": "",
+
+        "Code available (boolean)": false,
+        "Code URL": "",
+        "Code URL2": "",
+        "MD5 sum (for archives)": "",
+        "MD5 sum (for archives) URL2": "",
+        "git/hg/svn commit hash or revision number": "",
+        "git/hg/svn commit hash or revision number URL2": "",
+
+        "Software Heritage permalink": "",
+
+
+        "If code not available, pseudo-code available (boolean)": true,
+
+        "If pseudo-code, could the paper be trivially implemented? {0..4}": 2,
+
+        "Software type {Code, Binary, Partial Code}": "",
+
+        "Software language": "c++",
```

Comment:
```suggestion
        "Software language": "",
```
it does not make sense to have a language and no code provided ;)

Reply:
Thanks @pbourqua
shall we commit this change and approve ?